app/src/main/res/layout/migrate_wallet_error_fragment.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
could this not be merged with create_wallet_error_fragment.xml ?

Reply:
👍